modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1116 @@
+
+For information on the **A11ychecker** plugin, see: xref:a11ychecker.adoc[A11ychecker plugin].
+
+
+=== Comments 3.1.0
+
+The {productname} 6.1 release includes an accompanying release of the **Comments** premium plugin.
+
+NOTE: Enabling any of these features depends on changes made to the Core {productname} editor. As a consequence, version 3.1.0 of the xref:comments.adoc[Comments Premium Plugin] requires {productname} 6.1 or later. 
+
+**Comments** 3.1.0 introduces the following enhancements:
+
+
+==== A new event log API
+
+A new API, `editor.plugins.tinycomments.getEventLog()` is now available.
+
+As its name suggests, it returns a log of all Comment-related events, including:
+
+* when a new comment is added (and who added it).
+
+* when a comment is edited (and who edited it).
+
+* when a reply to a comment is added (and who added the reply).
+
+* when a comment is resolved (and who resolved the comment).
+
+* when a comment is deleted (and who deleted the comment).
+
+The new API also supports an `after` option.
+
+[source,js]
+----
+editor.plugins.tinycomments.getEventLog(
+  { after: '<iso-8601-format-time-stamp>' }
+)
+----
+
+restricts the returned list to Comment events after the time-stamp date.
+
+Underlying this new API is a new `CommentChange` event, which gets fired when any modification is made to Comments data.
+
+This new event also exposes the `getEventLog()` API. For example:
+
+[source,js]
+----
+editor.on('CommentChange', (e) => {
+  const logs = e.getEventLogs();
+});
+----
+
+
+==== Avatars can now be displayed next to usernames in comments
+
+{productname} now generates a different default avatar icon for each user.
+
+And this default icon, presented next to user names in comments, can be replaced with a custom avatar.
+
+In embedded mode, the avatar is supplied by the `tinycomments_author_avatar` option:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea', // change this value according to your HTML
+  plugins: 'tinycomments',
+  tinycomments_mode: "embedded",
+  tinycomments_author: "testuser",
+  tinycomments_author_name: "Test User",
+  tinycomments_author_avatar: "https://example.com/image.ext"
+});
+----
+
+In callback mode, the avatar is supplied via the `tinycomments_create` and `tinycoments_reply` callbacks.
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea', // change this value according to your HTML
+  plugins: 'tinycomments',
+  tinycomments_mode: 'callback',
+  tinycomments_create: create_comment, // Add the callback to TinyMCE
+  tinycomments_reply: reply_comment,   // Add the callback to TinyMCE
+  tinycomments_edit_comment: edit_comment,
+  tinycomments_delete: delete_comment_thread,
+  tinycomments_delete_all: delete_all_comment_threads,
+  tinycomments_delete_comment: delete_comment,
+  tinycomments_lookup: lookup_comment,
+  tinycomments_author: "testuser",
+  tinycomments_author_name: "Test User",
+  tinycomments_author_avatar: "https://example.com/image.ext"
```

Comment:
These 3 options only relate to embedded mode so we probably don't want to include them in a callback mode example.